pkgs/development/python-modules/matplotlib/default.nix

```diff
@@ -172,28 +176,49 @@ buildPythonPackage rec {
     ++ lib.optionals enableTk [ tkinter ];
 
   passthru.config = {
-    directories = {
-      basedirlist = ".";
-    };
-    libs = {
-      system_freetype = true;
-      system_qhull = true;
-      # LTO not working in darwin stdenv, see #19312
-      enable_lto = !stdenv.isDarwin;
-    };
+    # In previous version v3.8.4 we used this passthru.config to generate an
+    # INI build configuration file. Now upstream switched to meson, so we
+    # retain sort of compatibility with the attribute set we had in the past,
+    # and we generate the `mesonFlags` from this attribute set.
```

Comment:
We don't really need to preserve those, especially if the names are not the same and it takes a 4 line comment to explain it.

Reply:
> We don't really need to preserve those, especially if the names are not the same and it takes a 4 line comment to explain it.

Hmm yes I guess you are right.. hopefully no body relied upon that config outside of Nixpkgs.